dev/com.ibm.ws.jaxrs.2.1.sse_fat/publish/servers/com.ibm.ws.jaxrs21.sse.basic/server.xml

```diff
@@ -19,5 +19,7 @@
 	<variable name="onError" value="FAIL" />
 
 	<application location="BasicSseApp.war"/>
+	
+	<javaPermission className="java.security.AllPermission"/>
```

Comment:
I really don't like this. Can you change it so that it points out each permission that it needs?  And since the 2.1 version didn't need any permission, I'm guessing that this means that there is a RESTEasy class that is doing something that should be in a doPriv.  If that is the case, can you open a JIRA and then put the link to that JIRA here?

Reply:
Agreed.  This is only temporary and will be addressed before merging.

Reply:
No longer needed.